templates/zerver/emails/deactivate.source.html

```diff
@@ -0,0 +1,16 @@
+{% extends "zerver/emails/compiled/email_base_default.html" %}
+
+{% block illustration %}
+<img src="{{ email_images_base_uri }}/email_logo.png" alt=""/>
+{% endblock %}
+
+{% block content %}
+Dear {{username}},
+<br/><br/>
+Your Zulip account on {{realm_name}} has been deactivated, and you will no longer be able to log in.
+<br/><br/>
+The administrator for {{realm_name}} sent the following message:
+
+<pre class="deactivated-user-text">{{ text }}</pre>
```

Comment:
Why is this a `pre` block?  I would have thought a `blockquote` would be more appropriate for what we're doing (might require less CSS).

Reply:
Because otherwise linebreaks for example are not recognized.

Reply:
OK, we can use blockquote with just this line added:
```
 diff --git a/templates/zerver/emails/email.css b/templates/zerver/emails/email.css
index 4b554cea3d..ef78febd8e 100644
--- a/templates/zerver/emails/email.css
+++ b/templates/zerver/emails/email.css
@@ -332,6 +332,7 @@ a.button:hover {
     padding: 0 0 0 15px;
     margin: 0 0 20px;
     border-left: 5px solid #eee;
+    white-space: pre-line;
 }
 
 @media only screen and (max-width: 620px) {
```